styles/main.css

```diff
@@ -97,10 +98,11 @@ header nav a {
     /* margin-left: 1em; */
     margin-right: 2em;
     display: block;
-}
-header a:hover {
     border-top-style: solid;
     border-width: 4px;
+    border-color: transparent;
+}
```

Comment:
These styles should apply to  `header nav a` as a default for non-hovered header links